Drawing/Drawing.js

```diff
@@ -75,13 +100,100 @@ const DrawingBase = kind({
 		 * @default false
 		 * @public
 		 */
-		disabled: PropTypes.bool
+		disabled: PropTypes.bool,
+
+		/**
+		 * Overrides the default Drawing control component to support customized behaviors.
+		 *
+		 * The provided component will receive the following props from `Drawing`:
+		 *
+		 * * `brushColor` - Color used for brush
+		 * * `brushSize` - Value used for brush size
+		 * * `canvasColor` - Color of canvas
+		 * * `disabled` - Disables drawing controls
+		 * * `fillColor` - Color used when `drawingTool=fill`
+		 * * `setBrushColor` - Called when `brushColor` is changed
+		 * * `setBrushSize` - Called when `brushSize` is changed
+		 * * `setDrawingTool` - Called when `drawingTool` is changed
+		 * * `setFillColor` - Called when `fillColor` is changed
+		 *
+		 * @type {Component|Element}
+		 * @default sandstone/Drawing.DrawingControls
+		 * @public
+		 */
+		drawingControlsComponent: EnactPropTypes.componentOverride,
+
+		/**
+		 * Sets the tool of drawing.
+		 *
+		 * @type {String}
+		 * @default brush
+		 * @public
+		 */
+		drawingTool: PropTypes.string,
+
+		/**
+		 * Overrides the default Drawing utils component to support customized behaviors.
+		 *
+		 * The provided component will receive the following props from `Drawing`:
+		 *
+		 * * `backgroundImage` - Sets an image as canvas background
+		 * * `disabled` - Disables drawing utils
+		 * * `drawingRef` - Reference of drawing
+		 * * `fileInputHandler` - Called when user selects an image as canvas background
+		 * * `setBackgroundImage` - Called when the background of canvas is changed
+		 *
+		 * @type {Component|Element}
+		 * @default sandstone/Drawing.DrawingUtils
+		 * @public
+		 */
+		drawingUtilsComponent: EnactPropTypes.componentOverride,
+
+		/**
+		 * Sets the color of fill.
+		 *
+		 * @type {String}
+		 * @default #D0BB22
+		 * @public
+		 */
+		fillColor: PropTypes.string,
+
+		/**
+		 * Displays the drawing controls.
+		 *
+		 * When `true`, the drawing controls is displayed.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		showDrawingControls: PropTypes.bool,
+
+		/**
+		 * Displays the drawing utils.
+		 *
+		 * When `true`, the drawing utils is displayed.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		showDrawingUtils: PropTypes.bool
 	},
 
 	defaultProps: {
+		brushColor: '#545BCC',
+		brushSize: 5,
+		canvasColor: '#FFFFFF',
 		canvasHeight: 800,
 		canvasWidth: 1200,
-		disabled: false
+		disabled: false,
+		drawingControlsComponent: DrawingControls,
+		drawingTool: 'brush',
+		drawingUtilsComponent: DrawingUtils,
+		fillColor: '#D0BB22',
+		showDrawingControls: false,
+		showDrawingUtils: false
```

Comment:
Can we have true as default value? Or is something stopping us from having that?

Reply:
Sure. Having them as `true` seems more relevant